gub/3/new.py

```diff
@@ -1,4 +1,4 @@
-# https://docs.python.org/3.0/whatsnew/3.0.html
+# http://docs.python.org/3.0/whatsnew/3.0.html
```

Comment:
You're right on that one.... had to look a bit into git to see how to fix it, but with your hints and git's docu I think I managed to cleanup that part properly